tests/test_unit/approval_client/config.py

```diff
@@ -775,4 +779,19 @@
             ],
         },
     ),
+    APIResource(
+        api_name="migration.view_source",
+        resource_cls=ViewSource,
+        list_cls=NodeList[ViewSource],
+        _write_cls=ViewSourceApply,
+        _write_list_cls=NodeApplyList,
+        methods={
+            "create": [Method(api_class_method="upsert", mock_class_method="create_nodes")],
+            "delete": [Method(api_class_method="delete", mock_class_method="delete_id_external_id")],
+            "retrieve": [
+                Method(api_class_method="list", mock_class_method="return_values"),
+                Method(api_class_method="retrieve", mock_class_method="return_values"),
+            ],
+        },
+    ),
 ]
```

Comment:
This is updating the ApprovalClient we use in Toolkit for testing. Essentially simulating the ToolkitClient so we can intercept all CDF calls. For more information https://github.com/cognitedata/toolkit/tree/main/tests#approvalcogniteclient